src/colab_superarchives/views.py

```diff
@@ -407,6 +408,9 @@ def check_list_membership(self, user, mailinglist_name):
 
 
 class ManageUserSubscriptionsView(UserProfileBaseMixin, DetailView):
+    # Set the desired quantity of lists per page
+    LISTS_PER_PAGE = 10
```

Comment:
@matheus-fga , I modified this line so it can be set in 'settings.py', i.e.: https://github.com/colab/colab-superarchives-plugin/blob/master/src/colab_superarchives/views.py#L410